IGI.Lab2/src/CarPark.DAL/EF/CarParkContext.cs

```diff
@@ -1,20 +1,29 @@
 ﻿using CarPark.DAL.Models;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
 
 namespace CarPark.DAL.EF
 {
-    public class CarParkContext : DbContext
+    public class CarParkContext : IdentityDbContext<ApplicationUser>
     {
-        public CarParkContext(DbContextOptions<CarParkContext> options)
+        private readonly ILoggerFactory _loggerFactory;
+        public CarParkContext(DbContextOptions<CarParkContext> options, ILoggerFactory loggerFactory)
              : base(options)
         {
-            Database.EnsureCreated();
+            _loggerFactory = loggerFactory;
+            Database.Migrate();
```

Comment:
Why did you changed this method?

Reply:
Changed the context under Identity. Tables were not created right away, I experimented there and stayed on migrate (). Plus, a logger was added to the context.


This is not a mistake?

Reply:
I think migrate works only when db exist.
EnsureCreated still should be.

Reply:
Is it possible to leave both this and that? or EnsureCreate () only?

Reply:
fix

Reply:
I don't see any of your fixes.
Update the branch.

Reply:
fix